utils.py

```diff
@@ -1022,30 +1022,40 @@ def get_fake_context(context=None, area_type='VIEW_3D'):
 
 
 def has_url(text):
-    # urls = re.findall('https?://(?:[-\w.]|(?:%[\da-fA-F]{2}))+', text)
-    urls = re.findall(r'(?<=<a href=")[^"]*', text)
-    urls_with_str = []
+    #first remove markdown *
+    text = text.replace('*','')
```

Comment:
We agreed about using double quotes as our standard for strings. I would change every string in the PR to double quotes.

Reply:
done